solidity/contracts/bridge/Bridge.sol

```diff
@@ -1312,8 +1310,87 @@ contract Bridge is Ownable, EcdsaWalletOwner {
             processedRedemptionsCount > 0,
             "Redemption transaction must process at least one redemption"
         );
+    }
+
+    /// @notice Processes a single redemption transaction output. Tries to
+    ///         identify output as a pending redemption request or reported
+    ///         redemption timeout. Output script passed to this function must
+    ///         not be the change output. Such output needs to be identified
+    ///         separately before calling this function.
+    ///         Reverts if outut is neither requested pending redemption nor
+    ///         requested and reported timed-out redemption.
+    ///         This function also marks each request as processed by removing
```

Comment:
`This function also marks each request` -> `This function also marks each pending request` to make it explicit?

Reply:
027b0bab7715569acf2430189f0b3bf1c14ec3d4